daml-script/export/integration-tests/reproduces-transactions/test/scala/com/daml/script/export/ReproducesTransactions.scala

```diff
@@ -72,13 +73,22 @@ final class ReproducesTransactions
     )
 
   private def collectTrees(client: LedgerClient, parties: List[Ref.Party]) =
-    client.transactionClient
-      .getTransactionTrees(
-        LedgerOffset().withBoundary(LedgerOffset.LedgerBoundary.LEDGER_BEGIN),
-        Some(LedgerOffset().withBoundary(LedgerOffset.LedgerBoundary.LEDGER_END)),
-        transactionFilter(parties: _*),
+    client.v2.updateService
```

Comment:
The updateService uses Participant offsets, instead of ledger offsets. I've updated the export tool to use participant offsets throughout, but it might be better to translate from ledger offsets to participant offsets when we call the api.
I'm not entirely sure what the difference is, or what you would want for this tool.